test/Microsoft.Framework.Runtime.Tests/FileGlobbing/ProjectFilesCollectionFacts.cs

```diff
@@ -111,6 +111,22 @@ public void RewriteCompileBuiltIn()
             Assert.Equal(NormalizePatterns("*.cs", "../*.cs", "**/*.cpp", "**/*.h"), target.CompilePatternsGroup.IncludePatterns);
         }
 
+
+        [Fact]
+        public void ExceptionThrowWhenWildcardPresentsInLiteralPath()
+        {
+            var rawProject = Deserialize(@"{""compileFiles"": ""*.cs""}");
+
+            var exception = Assert.Throws<FileFormatException>(() =>
+            {
+                var target = new ProjectFilesCollection(rawProject, string.Empty, string.Empty);
+            });
+
+            Assert.Equal(
+                "Pattern *.cs is not expected to contain wildcard characters.",
```

Comment:
Isn't this exception message supposed to be the same as the one on line 65 in `PatternsCollectionHelper` ?